psi4/driver/schema_wrapper.py

```diff
@@ -586,6 +586,16 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
             )):
                 json_data["extras"]["qcvars"][k] = _serial_translation(v, json=json_serialization)
 
+    # Add in handling of matrix arguments which need to be obtained by a
+    # a function call.
+    if "tIJAB" in json_data["extras"]["qcvars"]:
+        json_data["extras"]["qcvars"]["tIJAB"] = wfn.get_amplitudes("tIJAB").to_array()
+    if "tIA" in json_data["extras"]["qcvars"]:
+        json_data["extras"]["qcvars"]["tIA"] = wfn.get_amplitudes("tIA").to_array()
```

Comment:
```suggestion
    if "tIJAB" in json_data["extras"]["psi4:tamps"]:
        json_data["extras"]["psi4:tamps"]["tIJAB"] = wfn.get_amplitudes("tIJAB").to_array()
    if "tIA" in json_data["extras"]["psi4:tamps"]:
        json_data["extras"]["psi4:tamps"]["tIA"] = wfn.get_amplitudes("tIA").to_array()
```
I'm worried about extending `extras.qcvars` with extra fields because the routines that write/read from there think they have full control to delete it and fill it. Better to carve out your own bit of extras. `"psi4.tamps"` is just a thought.